src/sbt-test/dumpLicenseReport/default-report/example.sbt

```diff
@@ -7,11 +7,11 @@ excludeDependencies += SbtExclusionRule(organization = "org.scala-lang")
 
 TaskKey[Unit]("check") := {
   val contents = sbt.IO.read(target.value / "license-reports" / "example-licenses.md")
-  if (!contents.contains("[The Apache Software License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0.txt) | com.fasterxml.jackson.core # jackson-databind # 2.5.4"))
```

Comment:
do you have to change this from databind to core? I believe that databind was chosen specifically for this test